routers/admin.py

```diff
@@ -204,12 +204,14 @@ def _revoke_group_membership(
     reason: str | None,
     admin_record: BiocommonsUser,
     db_session: Session,
+    client: Auth0Client,
 ) -> None:
     membership = GroupMembership.get_by_user_id_and_group_id_or_404(
         user_id=user_id,
         group_id=group.group_id,
         session=db_session,
     )
+    membership.revoke_auth0_role(auth0_client=client)
```

Comment:
Do we need to check if this actually went through here? If the status is not `APPROVED` the role won't be removed.